test/unit/mailer_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+class MailerTest < ActiveSupport::TestCase
+  context "sending mail for mfa recommendation announcement" do
+    setup do
+      @user = create(:user)
+      Gemcutter::Application.load_tasks
+      Rake::Task["mfa_policy:announce_recommendation"].invoke
+      Delayed::Worker.new.work_off
+    end
+
+    should "send mail to users" do
```

Comment:
Is there a preference for testing mailers -- integration vs. unit tests?

This is currently set up as an integration test. We're testing the mailer through triggering the `mfa_policy:announce_recommendation` rake task.

We can revise to a unit test:
```ruby
require "test_helper"

class MailerTest < ActiveSupport::TestCase
  context "sending mail for mfa recommendation announcement" do
    setup do
      @user = create(:user)
    end

    should "send mail to users" do
      email = Mailer.mfa_recommendation_announcement(@user.id)

      assert_equal [@user.email], email.to
      assert_equal ["no-reply@mailer.rubygems.org"], email.from
      assert_equal "Official Recommendation: Enable multi-factor authentication on your RubyGems account", email.subject
      assert_match "Today, we've announced our security-focused ambitions to the community.", email.text_part.body.to_s
    end
  end
end